packages/fx-core/src/common/samples-config.json

```diff
@@ -1,6 +1,6 @@
 {
-    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/v1.0.0/",
-    "defaultPackageLink": "https://github.com/OfficeDev/TeamsFx-Samples/archive/refs/tags/v1.0.0.zip",
+    "baseUrl": "https://github.com/OfficeDev/TeamsFx-Samples/tree/v1.1.0/",
+    "defaultPackageLink": "https://github.com/OfficeDev/TeamsFx-Samples/archive/refs/tags/v1.1.0.zip",
```

Comment:
Why do we upgrade the sample version?

Reply:
The sample version is updated by spring release, not by me.

Reply:
Then you may need to clean up your commits. It's better if your PR only includes your changes.

Reply:
Thanks. Updated.